README.md

```diff
@@ -12,4 +12,31 @@
     
     - Ensure you have node.js installed
     - code build - gulp
-    - test run - gulp test
\ No newline at end of file
+    - test run - gulp test
+    
+# To package
+    
+    - Run 'gulp'.
+    - Run 'mvn package'.
+    - Clean node modules and restore production packages with 'npm install --production'.
+    - Copy node_modules to build directory.
```

Comment:
Regarding step 3, 4. These steps were not required when we developed the extension, during that time agent used to have vsts-task-lib shipped along with it and hence we didn't package in the task folder But now agent is not shipping any predefined modules and hence we need to ship this module by our self.  Thanks for bringing this into our notice I will add a backlog item to address this. After that simply gulp should do needful. 

For now this change can be checked in, if anyone is trying to contribute it will help them.